integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/WebhookSiteSteps.java

```diff
@@ -54,4 +69,18 @@ public void webhookSiteWithIdDoesNotContainsRequest(String webhookId, String req
                     .noneMatch(requestContent -> requestContent.contains(requestTextWithoutPlaceholders));
         }
     }
+
+    private boolean assertHeadersInRequest(WebhookSiteRequest request, DataTable parametersDatatable) {
+        Map<String, List<String>> headers = request.getHeaders();
+        parametersDatatable.asMap().forEach((key, value) -> {
+            String valueWithoutPlaceholders = ContextResolver.resolveWithScenarioContext(context, value);
+            assertThat(headers.get(key)).isNotNull();
```

Comment:
You can't use assertions in boolean method, in this case it will never return false, either return true or thows assertion error.